packages/wxt/src/storage.ts

```diff
@@ -242,17 +276,33 @@ function createStorage(): WxtStorage {
       const driver = getDriver(base);
       await driver.restoreSnapshot(data);
     },
-    watch: (key, cb) => {
-      const { driver, driverKey } = resolveKey(key);
-      return watch(driver, driverKey, cb);
-    },
+    watch: ((
+      keyOrWatchers:
+        | StorageItemKey
+        | Record<StorageItemKey, WatchCallback<any>>,
+      cb?: WatchCallback<any>,
+    ): any => {
```

Comment:
Hmm, we use arrays everywhere else... Using an object here breaks that consistency. Not opposed to this, but let me think about it.

Reply:
Hmm, but how would you watch multiple keys at once if you were to pass an array? Since you have to tell the watch what you want to watch which wouldn't really work nicely if you had to pass an array of arrays. I could make it a separate function `watchMany` or so if you like?